content/en/users/compute/orchestration/dodas/_index.md

```diff
@@ -1,13 +1,36 @@
 ---
 title: Dynamic On-Demand Analysis Software
 linkTitle: Dynamic On-Demand Analysis Software
-draft: true
 type: docs
 weight: 40
 description: >
-  Subtitle here
+  Instantiate on-demand customizable infrastructures for data analysis
 ---
 
 ## What is it?
 
-TBD
+DODAS enables the execution of user analysis code both in batch mode and
+interactively via the Jupyter interface. DODAS is highly customizable and offers
+several building blocks that can be combined together in order to create the
+best service composition for a given use case. The currently available blocks
+allow to combine Jupyter and HTCondor as well as Jupyter and Spark or simply a
+jupyter interface. In addition, they allow the management of data via caches to
+optimize the processing of remote data. This can be done either via XCache or
+MinIO S3 object storage capabilities. DODAS is based on docker containers and
+the related orchestration relies on Kubernetes that enables the possibility to
+compose the building blocks via a web-based user interface thanks to Kubeapps.
+
+In order to deploy it services over various backend, DODAS relies on
```

Comment:
The PaaS link is broken. Also, do not link to a draft page from a non-draft page.